### PR TITLE
DOC: Improve usage instrucion

### PR DESCRIPTION
**Pull request by pchoisel**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,5 @@ Data
 env
 __pycache__
 .vscode
+dicom_anonymizer.egg-info
+dist
```
[the remark this answers is not in the document]
Comment (pchoisel) [reply .gitignore]: Is that important ?

**Comment by pchoisel** [comment]
@finetjul

File: .gitignore (hunk 1 above)
[the remark this answers is not in the document]
Comment (finetjul) [reply .gitignore:8]: not important. 
There is usually a hook to prevent missing EOL for last line...

**Review by finetjul — APPROVED** [review]
LGTM

**Pull request merged**